practice/src/css/bases/index.css

```diff
@@ -1,3 +1,4 @@
 @import url(reset.css);
 @import url(global.css);
-@import url(typography.css);
\ No newline at end of file
+@import url(typography.css);
+@import url(variable.css);
```

Comment:
typography should import under variable 

```
@import url(typography.css);
@import url(variable.css);
```

I see you have defined the variable for the font (font name and font-weight). Please update variable for typography